### ISSUE TITLE
Doesn't work on OS X High Sierra

### ISSUE THREAD

**Issue by theogravity** [haensl/openssl-certgen#9]
```
# Already ran 'brew tap haensl/haensl'

$ brew -v
Homebrew 1.9.1
Homebrew/homebrew-core (git revision e5a7; last commit 2019-01-14)

$ brew install openssl-generate-certificates
Updating Homebrew...
==> Installing openssl-generate-certificates from haensl/haensl
==> Downloading https://github.com/haensl/openssl-certgen/archive/v1.2.0.tar.gz
Already downloaded: /Users/t.gravity/Library/Caches/Homebrew/downloads/eb834f600d667130d7cdf47e4ce19a2424055edb77c9cec781df34b2cdaec3d9--openssl-certgen-1.2.0.tar.gz
==> make install
Last 15 lines from /Users/t.gravity/Library/Logs/Homebrew/openssl-generate-certificates/01.make:
2019-01-13 16:23:38 -0800

make
install

uname: illegal option -- o
usage: uname [-amnprsv]
ln: /usr/bin/openssl-generate-certificates: Operation not permitted
make: *** [install] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/haensl/homebrew-haensl/issues
```

**Comment by haensl** [comment]
Hi theogravity,

thank you for reporting this bug! I will look into it as soon as I find some time!

**Comment by haensl** [comment]
The `uname` error should be fixed in v1.2.2. You have to `brew update` in order to fetch the latest version.
Unfortunately the `Operation not permitted` error is not caused by the formula, but by OSX, see [https://gist.github.com/irazasyed/7732946](https://gist.github.com/irazasyed/7732946).
